website/docs/workloads/specview/specview-profiles.md

```diff
@@ -10,7 +10,7 @@ Runs the stock SPECviewperf Workloads.
 
 * **Supported Platform/Architectures**
   * win-x64
-  * AMD v620 GPU
+  * AMD v620 GPU, AMD mi25 GPU
```

Comment:
Just for my education, does this workload work without a GPU? on Windows

Reply:
Yes, SPECview runs on machines without a GPU. However, benchmarking can take a very long time on such systems